desktop/.eslintrc.js

```diff
@@ -5,54 +5,18 @@ module.exports = {
         "lib/**/*",
         "build/**/*",
         "python/**/*",
```

Comment:
Should we add leading slashes to /build, /python, /lib, etc?

Reply:
Done. It also makes me realize that `**/node_modules/**/*` is extraneous. Removed as well.